front-end/src/renderer/services/electronUtilsService.ts

```diff
@@ -56,6 +58,24 @@ export const showOpenDialog = async (
   }
 };
 
+export async function sha384(str: string): Promise<string> {
```

Comment:
Just asking, do you want this to be the exact error message, if not, use the `commonIPCHandler`.
Again, it is just a question